scripts/sanitycheck

```diff
@@ -1786,9 +1783,9 @@ class TestSuite:
                         eleTestsuite.remove(tc)
 
             if not goal.failed and goal.qemu:
-                    qemu_time = "%s" %(goal.metrics["qemu_time"])
+                    handler_time = "%s" %(goal.metrics["handler_time"])
```

Comment:
I would take a chance for PEP8 fix here and below: space from both sides of `%` operator, (i.e. `"%s" % (goal`)

Reply:
well be dealt with in another PR